Timelock/project/timelock-contract/src/utils.sw

```diff
@@ -0,0 +1,13 @@
+library utils;
+
+use std::{bytes::Bytes, hash::sha256};
+
+pub fn create_hash(
```

Comment:
hmm I don't see the reason for this utils function as it seems like we could just use the std function instead, but I don't see a problem keeping it.